lib/internal/Magento/Framework/Data/Collection/AbstractDb.php

```diff
@@ -217,7 +218,7 @@ public function getSize()
     {
         if ($this->_totalRecords === null) {
             $sql = $this->getSelectCountSql();
-            $this->_totalRecords = $this->getConnection()->fetchOne($sql, $this->_bindParams);
+            $this->_totalRecords = $this->_totalRecords ?? $this->getConnection()->fetchOne($sql, $this->_bindParams);
```

Comment:
It seems that when if reach this point, `$this->_totalRecords` must be strictly `null` at the time null coalescence check. Or am i missing something?

@orlangur @Den4ik

Reply:
@korostii Null coalescence check won't broke anything in this case, but looks strange :)
@p-bystritsky Do you remember what static tests you were fixed by this code?